### ISSUE TITLE
Does it make sense to have a value defined as constant and as distributed parmaeter for the same species

### ISSUE THREAD

**Issue by msevestre** [Open-Systems-Pharmacology/PK-Sim#2702]
Example: Gastric emptying time

![image](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/1041237/ba436dfa-29cb-4bb7-b99c-c2367801e58e)

and

![image](https://github.com/Open-Systems-Pharmacology/PK-Sim/assets/1041237/71eb14f1-9707-4b88-8e19-02e2585998ef)

We have code that removes one to add the other but....I can't help but feel we should not have a constant parameter in the first place. Or what am I missing? It's only distributed for a sub section of the populations? Weird

**Comment by Yuri05** [comment]
I think it does not make sense indeed; the entry for human should be removed from **tab_container_parameter_values**

**Comment by Yuri05** [comment]
currently, distribution "wins" over the const value  due to
https://github.com/Open-Systems-Pharmacology/PK-Sim/blob/4dda6236eb76e2f8ed2208584acf52f73ec91995/src/PKSim.Core/Services/ParameterContainerTask.cs#L183-L185

Could make sense if e.g. a parameter is defined as distributed just for 1 population and as a const value for all other populations.
But here it's defined for ALL population, so the const value does not make any sense.

**Comment by msevestre** [comment]
Yes it wins but I am wondering if we should remove it completely? Feels more like a db issue to me. This value is never used. Do we have parameters that are distributed for one population and constant for another?

**Comment by Yuri05** [comment]
ok, so looking at the (human) parameters which are currently present both as const values and distributions, we have 3 groups of parameters:

<details><summary>Group 1: just one mean_value and deviation=0 for all combinations of {population, gender}<br>     These parameters can be safely removed from distributions</summary>

container_id | container_type | container_name | parameter_name | default_value | number_of_pop_gender_combi | number_of_different_mean | max(standard_deviation)
-- | -- | -- | -- | -- | -- | -- | --
3950 | COMPARTMENT | Duodenum | Fraction mucosa | 0,26 | 2 | 1 | 0
3973 | COMPARTMENT | UpperJejunum | Fraction mucosa | 0,26 | 2 | 1 | 0
3996 | COMPARTMENT | LowerJejunum | Fraction mucosa | 0,26 | 2 | 1 | 0
4019 | COMPARTMENT | UpperIleum | Fraction mucosa | 0,13 | 2 | 1 | 0
4042 | COMPARTMENT | LowerIleum | Fraction mucosa | 0,13 | 2 | 1 | 0
4066 | COMPARTMENT | Caecum | Fraction mucosa | 0,19 | 2 | 1 | 0
4089 | COMPARTMENT | ColonAscendens | Fraction mucosa | 0,19 | 2 | 1 | 0
4112 | COMPARTMENT | ColonTransversum | Fraction mucosa | 0,21 | 2 | 1 | 0
4135 | COMPARTMENT | ColonDescendens | Fraction mucosa | 0,14 | 2 | 1 | 0
4158 | COMPARTMENT | ColonSigmoid | Fraction mucosa | 0,3 | 2 | 1 | 0
4181 | COMPARTMENT | Rectum | Fraction mucosa | 0,3 | 2 | 1 | 0
158 | ORGAN | LargeIntestine | Mucosa blood flow fraction | 0,75 | 2 | 1 | 0
168 | ORGAN | SmallIntestine | Mucosa blood flow fraction | 0,75 | 2 | 1 | 0
149 | ORGAN | Brain | Vf (lipid) | 0,11 | 2 | 1 | 0
149 | ORGAN | Brain | Vf (protein) | 0,081 | 2 | 1 | 0
149 | ORGAN | Brain | Vf (water) | 0,808 | 2 | 1 | 0

</details> 

<details><summary>Details</summary>

</details>
